minho/programmers/Level_1/2일차/가운데_글자_가져오기.js

```diff
@@ -0,0 +1,3 @@
+function solution(s) {
+    return s.length % 2 == 0 ? s.substr(s.length / 2 - 1, 2) : s.substr(s.length / 2, 1)
```

Comment:
```suggestion
    return s.length % 2 === 0 ? s.substr(s.length / 2 - 1, 2) : s.substr(s.length / 2, 1)
```
와우... 메서드 활용 너무 잘하셨어요